CanHazFunny/CanHazFunny/JokeService.cs

```diff
@@ -2,13 +2,14 @@
 
 namespace CanHazFunny;
 
-public class JokeService
+public class JokeService : IJokeServiceInterface
 {
     private HttpClient HttpClient { get; } = new();
 
     public string GetJoke()
     {
-        string joke = HttpClient.GetStringAsync("https://geek-jokes.sameerkumar.website/api").Result;
+        HttpResponseMessage json = HttpClient.GetAsync("https://geek-jokes.sameerkumar.website/api").Result;
+        string joke = json.Content.ReadAsStringAsync().Result;
```

Comment:
```suggestion
        string json = HttpClient.GetStringAsync("https://geek-jokes.sameerkumar.website/api?format=json").Result;
        var joke = JsonSerializer.Deserialize<JokeInformation>(json);
```
I think I would opt to use a JsonSerializer here